hermes/src/config.rs

```diff
@@ -37,7 +38,8 @@ type Result<T> = result::Result<T, Error>;
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Config {
     pub root_dir: PathBuf,
-    pub port: u16
+    pub port: u16,
+    pub network_names: HashSet<String>
```

Comment:
I am not sure it is worth to use such structure here, I don't see a use case for more than 2 network types: mainnet and testnet (maybe staging?). So it won't be more than 2 or 3 networks here.